natvis/pch.h

```diff
@@ -4,7 +4,10 @@
 #define NOMINMAX
 
 #include <windows.h>
+#pragma warning(push)
+#pragma warning(disable : 4471)
 #include <vsdebugeng.h>
```

Comment:
Is there a way to pull a newer version of this nuget package?  I tried specifying the latest but couldn't `nuget pull` after that.

Reply:
I've tried before, and there's been some weirdness. But I've been making some visualizer improvements and hope to do some updating as part of that work! For now, though, I'm a big fan of keeping PRs tightly scoped when possible. :)